sdk/core/azure-core-test/src/test_proxy_manager.cpp

```diff
@@ -243,6 +243,7 @@ void TestProxyManager::SetProxySanitizer()
   addSanitizer(SanitizerType::General, g_accountRegex, "account");
   addSanitizer(SanitizerType::Body, "client_secret=(?<clientsecret>[^&]+)", "clientsecret");
   addSanitizer(SanitizerType::Body, "client_id=(?<clientid>[^&]+)", "clientid");
+  addSanitizer(SanitizerType::Body, "(?:<SignedTid>)(.*)(?:</SignedTid>)", "signedtid");
```

Comment:
The regex pattern shared by the Java team is now:
```text
(?<=<UserDelegationKey>).*?(?:<SignedTid>)(.*)(?:</SignedTid>)
```

Reply:
just updated and also added Signed oid which just popped up